litmus-portal/cluster-agents/subscriber/pkg/types/kubeobject.go

```diff
@@ -0,0 +1,85 @@
+package types
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type PodInfo struct {
```

Comment:
We don't need to define all related k8s struct, you can directly use the struct from the official k8s repo.